.changeset/strong-months-grab.md

```diff
@@ -0,0 +1,9 @@
+---
+'astro': major
+---
+
+Updates Markdown page handling to no longer respond with `charset=utf-8` in the `Content-Type` header. This matches the rendering behaviour of other non-content pages.
```

Comment:
```suggestion
Changes the default character encoding handling for Markdown pages.

Markdown pages no longer respond with `charset=utf-8` in the `Content-Type` header. This matches the rendering behaviour of other non-content pages.
```
I'm wondering if we can get a strong (correct, of course!) statement for the intro here that's more plain language just to introduce the change?

Reply:
> Changes the default character encoding handling for Markdown pages.

I don't think the PR changes the default character encoding since we still add a `<meta charset="utf-8">` tag automatically if there's no layout (which is on a default setup). So I'm not sure this change is accurate here.

Reply:
I guess I was trying to get a changes HOW character encoding is handled (even though you can still get the same character encoding; you might just have to do something different to get it) by default in Markdown pages, not changes the encoding itself.

If there is a helpful, accurate line to that effect that makes sense, I think it sets up the longer explanation nicely. But, non-blocking! No objections and I'll let you make the call!

Reply:
I updated it now to match the upgrade guide header. Let me know what you think!

Reply:
Looks great!